README.md

```diff
@@ -375,46 +374,36 @@ EBS volume needs to be created as well. See [gitlab.tf.json.template.py] and the
 Once you've configured the project and your personal deployment or a shared 
 deployment deployment you intend to create, and once you manually provisioned 
 the shared cloud resources, it is time to provision the cloud infrastructure 
-for your deployment. Running
+for your deployment. Run 
 
 ```
-make terraform
+make deploy
 ```
 
-will display a plan and ask you to confirm it. Please consult the Terraform
-documentation for details. You will need to run `make terraform` once to set up
-your deployment and every time code changes define new cloud resources. The
-resources are defined in `….tf.json` files which in turn are generated from
-`….tf.json.template.py` files which are simple Python scripts containing the
-desired JSON as Python dictionary and list literals and comprehensions.
-
-## 3.3 Deploying lambda functions
-
-Once the cloud infrastructure for your deployment has been provisioned, you can
-deploy the project code into AWS Lambda. Running
+to prepare the lambda functions defined in the `lambdas` directory for 
+deployment via Terraform. Then run
 
 ```
-make deploy
+make terraform
 ```
 
-Will create or update AWS Lambda functions for each lambda defined in the
-`lambdas` directory. It will also create or update an AWS API Gateway to proxy
-the functions that act as web services. We call those functions *API lambdas*.
+which will display a plan and ask you to confirm it. Please consult the 
+Terraform documentation for details. You will need to run `make terraform` once 
+to set up your deployment and again to deploy any code changes.
 
-## 3.4 Provisioning stable API domain names
+Any time you wish to change the code running in the lambdas you will need to 
+run `make deploy` followed by `make terraform`. 
 
-The HTTP endpoint offered by API Gateway have somewhat cryptic and hard to
-remember domain names:
+Some Terraform configuration is generated by `make deploy`, but the rest is
+defined in `….tf.json` files which in turn are generated from 
+`….tf.json.template.py` files which are simple Python scripts containing the
+desired JSON as Python dictionary and list literals and comprehensions. Changes
+to these files require running `make terraform` again in order to update
+cloud infrastructure for the selected deployment.
 
-https://klm8yi31z7.execute-api.us-east-1.amazonaws.com/hannes/
+## 3.3 Finding deployment on the internet
```

Comment:
```suggestion
## 3.3 Locating service endpoints via DNS
```

Reply:
"Locating service endpoints via DNS" confuses me since we are talking about the indexer as well.

I'm going to use "Locating deployment endpoints via DNS"

Reply:
A deployment defines many endpoints, our REST APIs, AWS', Elasticsearch.

```suggestion
## 3.3 Locating REST API endpoints via DNS
```